docs/pre-commit-hooks.md

```diff
@@ -83,6 +83,18 @@ hooks:
     args: ["--dev", "-f", "requirements.txt", "-o", "requirements.txt"]
 ```
 
+## poetry-install
+
+The `poetry-install` hook calls the `poetry install --sync` command
+to make sure the installed dependencies match the packages defined in `poetry.lock`.
```

Comment:
... and this still describes `poetry install --sync` so maybe just say that "all locked packages are installed".